app/src/main/java/org/fossasia/openevent/general/event/EventViewHolder.kt

```diff
@@ -26,6 +28,15 @@ class EventViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
                     .into(itemView.eventImage)
         }
 
+        itemView.setOnClickListener{
+            val fragment = EventDetailsFragment()
+            val bundle = Bundle()
+            bundle.putLong("EVENT_ID", event.id)
+            fragment.setArguments(bundle)
```

Comment:
Send back the click to the fragment and let it handle this

Reply:
okay